apis/feature_extraction/inference/app/api.py

```diff
@@ -0,0 +1,42 @@
+import json
+import re
+from urllib.parse import unquote_plus
+import numpy as np
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+from .image import get_image_from_url
+from .lsh import LSHEncoder
+from .feature_extraction import extract_features
+
+lsh_encoder = LSHEncoder('2020-03-05')
+
+# initialise API
+app = FastAPI(
+    title='Feature Vector',
+    description='Takes an image url and returns the image\'s feature vector encoded as an LSH string',
+    docs_url='/feature-vector/docs',
+    redoc_url='/feature-vector/redoc'
+)
+
+
+@app.get('/feature-vector/')
+def feature_similarity_by_catalogue_id(image_url: str):
```

Comment:
Perhaps there could be another parameter, `iiif_image_info_url`, which can be used when dealing with IIIF images?